docs/GettingStarted.md

```diff
@@ -98,6 +98,8 @@ react-native run-windows
 
 A new Command Prompt window will open with the React packager as well as a React Native Windows app. You can now start developing! :tada:
 
+*Note:* If you want to work on another PC, you have to import the pfx file on the PC by opening `windows/<AppName>/<AppName>_TemporaryKey.pfx` and by entering `password` as the required password.
```

Comment:
@rozele I think the reason is described in this PR: https://github.com/Microsoft/react-native-windows/pull/1141#comment-120633438

Reply:
That PR comment also acknowledges that we could generate a cert for a domain, so that might be a better solution since most users will test across devices on the same domain?

Reply:
Hmm, in my case, I prefer password to [usernames or group names](https://docs.microsoft.com/en-us/powershell/module/pkiclient/export-pfxcertificate). For example, it's difficult to know usernames or group names in OSS development. (No password/protection is ideal.)

Reply:
We just ran into this exact thing and just guessed around to get it working. So +1 please add